datalad/crawler/nodes/misc.py

```diff
@@ -46,26 +46,29 @@ def __init__(self, keys=None, output=None, ignore_prefixes=['datalad_']):
           data
         ignore_prefixes : list, optional
           Keys with which prefixes to ignore.  By default all 'datalad_' ignored
+
         """
         self.data = []
         self.keys = keys
         self.output = output
         self.ignore_prefixes = ignore_prefixes or []
 
-    def get_values(self, *keys):
+    def get_values(self, keys):       # need to fix relevant usages into lists!!
```

Comment:
using TODO:  in your comments would make pycharm "mark" that line which would bring your attention